### PR TITLE
[Validation] Not blank for messages

### PR DESCRIPTION
**Pull request by Arminek**

### CODE REVIEW

File: tests/Behat/Context/UI/AdministratorOrderCommentsContext.php [hunk 1]
```diff
@@ -68,7 +78,16 @@ public function thisOrderShouldHaveACommentWithFromThisAdministrator(string $mes
     }
 
     /**
+     * @Then I should be notified that comment is invalid
+     */
+    public function thisOrderShouldNotHaveEmptyCommentFromThisCustomer(): void
+    {
+        #TODO
```
Comment (lchrusciel) [inline tests/Behat/Context/UI/AdministratorOrderCommentsContext.php:85]: Some longer comment why didn't we do it in a first place would be nice

File: tests/Comments/Infrastructure/Form/Type/OrderCommentTypeTest.php [hunk 2]
```diff
@@ -16,32 +16,34 @@
...
-        $form = $this->factory->create(OrderCommentType::class);
-
-        $form->submit(['message' => 'Hello']);
-
-        $comment = new OrderComment();
-        $comment->message = 'Hello';
-
-        $this->assertEquals($comment, $form->getData());
+        #TODO Cannot test form with constraints cuz symfony is bulwa
+//        $form = $this->factory->create(OrderCommentType::class);
```
Comment (lchrusciel) [inline tests/Comments/Infrastructure/Form/Type/OrderCommentTypeTest.php:20]: Let's just remove these tests then.

Anyway, would the problem remain if we will move validation definition to `validation.xml` file?

File: tests/Comments/Infrastructure/Form/Type/OrderCommentTypeTest.php [hunk 3]
```diff
@@ -16,32 +16,34 @@
...
     {
-        $form = $this->factory->create(OrderCommentType::class);
-
-        $form->submit(['message' => 'Hello']);
-
-        $comment = new OrderComment();
-        $comment->message = 'Hello';
-
-        $this->assertEquals($comment, $form->getData());
+        #TODO Cannot test form with constraints cuz symfony is bulwa
```
Comment (lchrusciel) [inline tests/Comments/Infrastructure/Form/Type/OrderCommentTypeTest.php:19]: Why is symfony the bulwa 🕺

**Review by lchrusciel — CHANGES_REQUESTED** [review]

File: tests/Comments/Infrastructure/Form/Type/OrderCommentTypeTest.php (hunk 2 above)
Comment (Arminek) [reply tests/Comments/Infrastructure/Form/Type/OrderCommentTypeTest.php:20]: Yes, It should solve the problem

**Pull request closed without merging**